File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/Storyboards.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class Storyboards{

	@SerializedName("playerStoryboardSpecRenderer")
	private PlayerStoryboardSpecRenderer playerStoryboardSpecRenderer;

	public void setPlayerStoryboardSpecRenderer(PlayerStoryboardSpecRenderer playerStoryboardSpecRenderer){
		this.playerStoryboardSpecRenderer = playerStoryboardSpecRenderer;
	}

	public PlayerStoryboardSpecRenderer getPlayerStoryboardSpecRenderer(){
		return playerStoryboardSpecRenderer;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/Storyboards.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import com.google.gson.annotations.SerializedName

class Storyboards {
    @SerializedName("playerStoryboardSpecRenderer")
    var playerStoryboardSpecRenderer: PlayerStoryboardSpecRenderer? = null
    override fun toString(): String {
        return "Storyboards{" +
                "playerStoryboardSpecRenderer = '" + playerStoryboardSpecRenderer + '\'' +
                "}"
    }
}